docs/website/docs/building-from-source/getting-started.md

```diff
@@ -138,32 +144,102 @@ cmake --build ../iree-build/
 
     See also our [developer documentation for ccache](https://github.com/openxla/iree/blob/main/docs/developers/developing_iree/ccache.md).
 
-## What's next?
+### Optional components
+
+By default, the CMake build includes:
+
+* All compiler targets (`llvm-cpu`, `cuda`, `vulkan-spirv`, etc.)
+* All runtime HAL drivers (`local-task`, `cuda`, `vulkan`, etc.)
+* All compiler input formats (StableHLO, TOSA, etc.)
+* All compiler output formats (VM bytecode, C)
 
-<!-- TODO(scotttodd): "at this point you can..." -->
+The default build does _not_ include:
+
+* Compiler or runtime bindings (Python, TFLite, etc.)
```

Comment:
I think that (perhaps in a followup), we should move the Python configuration to this page as an optional component. We can leave the other random integrations on an aux page, but Python support is a core supported features and should be here.

Reply:
I'd be curious how many developers/contributors actually need to build and edit the Python bindings from source. Probably more over time as JAX / PJRT integration picks up speed.

The environment setup steps on https://openxla.github.io/iree/building-from-source/python-bindings-and-importers/ are fairly complicated for casual users that want to follow quickstart instructions. Maybe the dev container can help there (configure the right version of Python by default).